apps/ThinQ_Integration.groovy

```diff
@@ -1066,7 +1066,7 @@ def decodeBinaryRTIMessage(protocol, modelInfo, data, returnCode) {
 				def associatedKey = targetKey.associatedAttributes.find {it == attribute}
 				if (associatedKey != null) {
 					def associatedKeyValue = output[associatedKey]
-					def newKey = targetKey."${associatedKey}".$"{targetKeyValue}"
+					def newKey = targetKey."${associatedKey}"."${targetKeyValue}"
```

Comment:
Good catch, did it actually work after changing this?

Reply:
Yes, it looks good so far.... this line was just a syntax error in the app that was preventing post execution in the device.  

Here are my values for my LG refrigerator model.  I am realizing that temp values are the SET values, not actual, since they never change...  So contact and icePlus may be the only attributes that have any merit... Do other LG models show actual temps?

**Current States**
contact : closed
freezerTemp : 0
freshAirFilterStatus : auto
fridgeTemp : 34
icePlus : off

Reply:
BTW, since the ThinQ app appropriately polls every minute for V1 devices, the chances that this device will report a 'contact' change is remote.  It is not worth it to poll more frequently, as I probably will not code for anything more than a repeated door open state for over a period of several minutes to trap for persistent door open state...

Ugh, LG put a V1 controller in my 2019 model...

Reply:
I have found no rhyme or reason as to which devices are v1 vs v2. I also haven't found any evidence that there is a way to upgrade, unfortunately.  They probably just cheap out and use whatever parts were left over in the parts bin :( To my knowledge, no, none of the fridges show the current temp. I agree that would be much more useful. Mine does just like yours, it's the setpoints, not the current temp reading. Kind of sucks, I wanted to originally build an alert like "fridge is too warm, something might be wrong" but no luck.

Reply:
I placed an old ST Zigbee contact sensor in our downstairs large capacity upright freezer that reports internal temperature to HE and keeps a decent battery level for about 6-8 months.  It is critical to know, in case of the freezer (filled to the brim with $$$$ meat and other items) if it goes above 5°F for more than 15 mins.  With an alert, I can quickly check it before it gets too warm and all the items are lost.  

For the LG, which has a smaller capacity freezer and less $ items, I'm thinking I'll wait and see what I want to do for real time temperature measurements.  LG really went cheap on this aspect, as reporting the set temps is weak, but having the real time temps of the inside, a real home automation need, especially with the $3K I paid for this 30 cubic foot deluxe model...